addons/purchase/models/purchase.py

```diff
@@ -658,18 +658,15 @@ def _send_reminder_mail(self, send_single=False):
                     order.with_context(is_reminder=True).message_post_with_template(template.id, email_layout_xmlid="mail.mail_notification_paynow", composition_mode='comment')
 
     def send_reminder_preview(self):
-        if not self.user_has_groups('purchase.group_send_reminder') and not self.receipt_reminder_email:
```

Comment:
I would leave the user_has_group anyway to avoid any side effect or overhide of this function

Reply:
ok. I'll keep this part.